io.qt.qt5-base.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
It was probably addressed during initial review but I will ask anyway: it requires both Qt and full GNOME runtime?

Reply:
It doesn't require the full GNOME runtime, but it's a GTK + some GNOME stuff GUI app, yes. Qt is required (at least for the foreseeable future) for `libringclient` which contains everything non-GUI that is shared between different client implementations. The dependency on `Qt5Gui` should be gone very soon however, it's only used in one place to load, scale and save avatar images.